src/identifier.js

```diff
@@ -23,10 +23,10 @@ Identifier.prototype.save = function() {
       value: this.value,
       visitor_id: this.visitorId
     })
-    .then(identifierJson => {
+    .then(({ data }) => {
```

Comment:
Can't wait until we have Typescript and we can catch these type of errors at compile time.